flink-core/src/main/java/org/apache/flink/configuration/RestartBackoffTimeStrategyOptions.java

```diff
@@ -54,4 +54,23 @@
 		.key("restart-backoff-time-strategy.failure-rate.backoff-time")
 		.defaultValue("1 min")
 		.withDescription("Backoff time between two consecutive restart attempts.");
+
+	/**
+	 * Maximum number of attempts the fixed delay restart strategy will try before failing a job.
+	 */
+	@PublicEvolving
+	public static final ConfigOption<Integer> RESTART_BACKOFF_TIME_STRATEGY_FIXED_DELAY_ATTEMPTS = ConfigOptions
+		.key("restart-backoff-time-strategy.fixed-delay.attempts")
+		.defaultValue(1)
+		.withDescription("Maximum number of attempts the fixed delay restart strategy will try before failing a job.");
+
+	/**
+	 * Backoff time between two consecutive restart attempts in FixedDelayRestartBackoffTimeStrategy.
+	 * It can be specified using Scala's FiniteDuration notation: "1 min", "20 s".
+	 */
+	@PublicEvolving
+	public static final ConfigOption<String> RESTART_BACKOFF_TIME_STRATEGY_FIXED_DELAY_BACKOFF_TIME = ConfigOptions
+		.key("restart-backoff-time-strategy.fixed-delay.backoff-time")
+		.defaultValue("1 min")
```

Comment:
I would be in favor of specifying the backoff time in milli seconds in order to get eventually rid of Scala's `FiniteDuration`.

Reply:
OK